app.js

```diff
@@ -0,0 +1,74 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const logger = require('morgan')
+
+require('dotenv').config()
+
+var PORT = process.env.PORT;
+
+var app = express()
```

Comment:
Use ES6 and always add a semicolon to a JavaScript expression

Reply:
done.